tests/lib/rules/lines-around-comment.js

```diff
@@ -317,8 +317,195 @@ ruleTester.run("lines-around-comment", rule, {
                 allowBlockEnd: true
             }],
             ecmaFeatures: {classes: true}
-        }
+        },
+
+        // check for object start comments
```

Comment:
Can you also a small code example for the code piece actually present inside the issue as it related to return.
Also can you please split the `code` into multi-line has its really difficult for the reviewer to read and understand.